lab_1_classify_by_unigrams/main.py

```diff
@@ -20,6 +21,14 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if not isinstance(text, str) or len(text) == 0:
+        return None
+
+    tokens = []
+    for symbol in text:
```

Comment:
Call lower only once for the source text

Reply:
done